docs/concepts.rst

```diff
@@ -116,6 +116,47 @@ DAGs can be used as context managers to automatically assign new operators to th
 
     op.dag is dag # True
 
+.. _concepts:functional_dags:
+
+Functional DAGs
+---------------
+*Added in Airflow 1.10.11*
+
+DAGs can be defined using functional abstractions. Outputs and inputs are sent between tasks using
+:ref:`XComs <concepts:xcom>` values. In addition, you can wrap functions as tasks using the
+:ref:`task decorator <concepts:task_decorator>`. Dependencies are automatically inferred from
+the message dependencies.
```

Comment:
"from the message dependencies" isn't quite clear - what it means. (What's a message? _I_ know what you mean, but we need to reword this)

Reply:
"Airflow will also automatically add dependencies between tasks to ensure that XCom messages are available when operators are executed."

wdyt?

Reply:
that `XCom values`, then SGTM.